packages/tooltip/test/lion-tooltip.test.js

```diff
@@ -25,6 +25,7 @@ describe('lion-tooltip', () => {
       const eventMouseEnter = new Event('mouseenter');
       invoker.dispatchEvent(eventMouseEnter);
       await el.updateComplete;
+      await aTimeout();
```

Comment:
this is very brittle implementation IMO

we have to find another way to wait till the work is done

Reply:
All aTimeout()s introduced by this PR can be removed fortunately :). They were needed I think when it was still loading on-demand instead of eagerly in constructor.